docs/en/latest/plugins/traffic-split.md

```diff
@@ -482,6 +483,98 @@ Content-Type: text/html; charset=utf-8
 hello 1980
 ```
 
+### Matching rules correspond to upstream
+
+How to make different matching rules correspond to different upstreams. We can achieve this by configuring multiple `rules.match` + `rules.weighted_upstreams`.
+
+**Example:**
+
+When the request header `x-api-id` is equal to 1, it hits the upstream of port 1981; when `x-api-id` is equal to 2, it hits the upstream of port 1982; otherwise, it hits the upstream of port 1980 (the upstream response data is the corresponding port number).
```

Comment:
"the upstream of port" => "the upstream with port"

Please fix other similar places.

Reply:
Updated.